packages/meditrak-server/src/routes/importSurveys/ConfigImporter/processCalculatedConfig.js

```diff
@@ -0,0 +1,195 @@
+/**
+ * Tupaia
+ * Copyright (c) 2017 - 2020 Beyond Essential Systems Pty Ltd
+ */
+
+import { getVariables } from '@beyondessential/arithmetic';
+import { splitStringOn, splitStringOnComma } from '../../utilities';
+
+const OPERATOR_TRANSLATIONS = {
+  GT: '>',
+  GE: '>=',
+  LT: '<',
+  LE: '<=',
+  EQ: '=',
+};
+
+export const processCalculatedConfig = async (models, config) => {
+  const calculatedConfig = await processCalculatedConfigByType(models, config);
+  return {
+    type: config.type,
+    ...calculatedConfig,
+  };
+};
+
+const processCalculatedConfigByType = async (models, config) => {
+  switch (config.type) {
+    case 'arithmetic':
+      return translateArithmeticConfig(models, config);
+    case 'condition':
+      return translateConditionConfig(models, config);
+    default:
+      throw new Error(`Invalid calculated type: ${config.type}`);
+  }
+};
+
+const translateArithmeticConfig = async (models, config) => {
+  const codes = getVariables(config.formula);
+
+  let translatedConfig = {
+    formula: await translateFormula(models, config.formula, codes),
+  };
+
+  if (config.defaultValues) {
+    translatedConfig = {
+      ...translatedConfig,
+      defaultValues: await translateDefaultValues(models, config.defaultValues),
+    };
+  }
+
+  if (config.valueTranslation) {
+    translatedConfig = {
+      ...translatedConfig,
+      valueTranslation: await translateValueTranslation(models, config.valueTranslation),
+    };
+  }
+
+  if (config.text) {
```

Comment:
```suggestion
  if (config.answerDisplayText) {
```
Or `answerLabel`. I worry that `text` gets confused with the `question.text`